swarm/activations.py

```diff
@@ -0,0 +1,10 @@
+#  -*- coding: utf-8 -*-
+
+
+import torch
+from torch.nn import *
+
+
+class xtanh(Module):
+    def forward(self, x: torch.Tensor):
+        return x - torch.tanh(x)
```

Comment:
I don't really get why xtanh is here and every other possible activation isn't? And ReLU doesn't have to be? (I haven't tried yet to understand this in detail either.)

Reply:
This is because of the `from torch.nn import *`. All the activation functions are also pulled in automatically from torch. `xtanh` is Ben's custom activation hence I defined it here